playbook/app/pb_kits/playbook/pb_table/table_col.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= content_tag(:col,
+    span: object.span,
+    aria: object.aria,
+    class: object.classname,
+    data: object.data,
+    style: "background-color: #{object.background_color};",
```

Comment:
PR looks great @markdoeswork ! Just had a question on this piece, any particular reason we're having to use a style tag here instead of applying the color via scss? I do see that you're adding classnames based on the chosen background color in the .rb file so wanted to know what the reasoning was here

cc: @jasperfurniss

Reply:
++

Reply:
background color is being handled by a method in the controller `def classname` that follows the same class naming convention as our background kit